### PR TITLE
Add, edit and delete service levels

### PR DESCRIPTION
**Pull request by bjss-colin**

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/ServiceLevelAgreements/ServiceLevelCredits.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace NHSD.GPIT.BuyingCatalogue.WebApp.Areas.Admin.Models.ServiceLevelAgreements
+{
+    public enum ServiceLevelCredits
+    {
+        No = 0,
+        Yes = 1,
+    }
+}
```
Comment (PhysicsJon) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/ServiceLevelAgreements/ServiceLevelCredits.cs:8]: Do we not have an extension that could do this and make the db column a bit?

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
A small change (look for BooleanExtensions)

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/ServiceLevelAgreements/ServiceLevelCredits.cs (hunk 1 above)
Comment (bjss-colin) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/ServiceLevelAgreements/ServiceLevelCredits.cs:8]: I initially added this for the frontend. Will take a further look to see if `bool` can instead be used for the radio list.

Comment (bjss-colin) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/ServiceLevelAgreements/ServiceLevelCredits.cs]: Removed this and switched to using bool

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**